website/docs/plugin/framework/handling-data/types/map.mdx

```diff
@@ -0,0 +1,127 @@
+* [`(types.Map).ElementsAs(context.Context, any, bool) diag.Diagnostics`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types/basetypes#MapValue.ElementsAs): Converts the known values into the given Go type, if possible. It is recommended to use a map of framework types to account for elements which may be unknown.
+
+In this example, a map of strings value is checked for being null or unknown value first, before accessing its known value elements as a `[]types.String`:
+
+```go
+// Example data model definition
+// type ExampleModel struct {
+//   ExampleAttribute types.Map `tfsdk:"example_attribute"`
+// }
+//
+// This would be filled in, such as calling: req.Plan.Get(ctx, &data)
+var data ExampleModel
+
+// optional logic for handling null value
+if data.ExampleAttribute.IsNull() {
+    // ...
+}
+
+// optional logic for handling unknown value
+if data.ExampleAttribute.IsUnknown() {
+    // ...
+}
+
+elements := make(map[string]types.String, len(data.ExampleAttribute.Elements()))
+diags := data.ExampleAttribute.ElementsAs(ctx, &elements, false)
+```
+
+## Setting Values
+
+Call one of the following to create a `types.Map` value:
+
+* [`types.MapNull(attr.Type) types.Map`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapNull): A null list value with the given element type.
+* [`types.MapUnknown(attr.Type) types.Map`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapUnknown): An unknown list value with the given element type.
+* [`types.MapValue(attr.Type, map[string]attr.Value) (types.Map, diag.Diagnostics)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapValue): A known value with the given element type and values.
+* [`types.MapValueFrom(context.Context, attr.Type, any) (types.Map, diag.Diagnostics)`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapValueFrom): A known value with the given element type and values. This can convert the source data from standard Go types into framework types as noted in the documentation for each element type, such as giving `map[string]*string` for a `types.Map` of `types.String`.
+* [`types.MapValueMust(map[string]attr.Type, map[string]attr.Value) types.Map`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/types#MapValueMust): A known value with the given element type and values. Any diagnostics are converted to a runtime panic. This is recommended only for testing or exhaustively tested logic.
+
+In this example, a known map value is created from framework types:
+
+```go
+elements := map[string]attr.Value{
+    "key1": types.StringValue("value1"),
+    "key2": types.StringValue("value2"),
+}
+listValue, diags := types.MapValue(types.StringType, elements)
```

Comment:
Nit: Could use `mapValue` to emphasise type in var.

Reply:
Good copypasta 🍝  catch, will update!